testing/internal/e2e/tests/static/session_cancel_admin_test.go

```diff
@@ -93,7 +94,37 @@ func TestCliSessionCancelAdmin(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, newTargetId, session.TargetId)
 	assert.Equal(t, newHostId, session.HostId)
-	require.Equal(t, "active", session.Status)
+
+	// Wait for session to be active
```

Comment:
Do we still need both this and the above retry loop? Can we not do both these in one retry?